site/pages/theme-topic-en.hbs

```diff
@@ -0,0 +1,127 @@
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+		</div>
+	</section>
+	<section class="gc-features mrgn-tp-xl">
```

Comment:
The h2 is missing.

This "gc-features" section feature are not designed to include and use a column for "follow us". The documented gc-features pattern need to be ajusted. May be something specific for one-column and two-column. And the three feature set may not be possible for this template.

Note that the "follow us" also introduce an header level 2.